assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -380,21 +382,37 @@ function BannerNotification( props ) {
 						</Fragment>
 					) : (
 						<p>
-							<span
-								dangerouslySetInnerHTML={ sanitizeHTML(
-									description,
-									{
-										ALLOWED_TAGS: [
-											'strong',
-											'em',
-											'br',
-											'a',
-										],
-										ALLOWED_ATTR: [ 'href' ],
-									}
-								) }
-							/>{ ' ' }
-							{ learnMoreAndPageIndex }
+							{ createInterpolateElement(
+								sprintf(
+									/* translators: Banner notification text. 1: Description in banner. 2: Learn more link. */
+									__( '%1$s %2$s', 'google-site-kit' ),
```

Comment:
I know that our main purpose of this PR is to make translation easier, but here I am not sure that we really translate anything. All what we do here is adding one more translatable string that offers people to translate a whitespace into all languages 😝 

This seems to be useless in my opinion. Let's probably revert it back to how it was?

Reply:
In this case the _order_ is still translatable. It's probably very unlikely, but theoretically a locale might want the link first (eg. `"Learn more about $description"` or something).

I know it's not a big deal, but in this case it's mostly about consistency. If we leave this one in as an exception, then someone uses this as a reference and copies it for a different translation string, it would set/serve as a bad example.

I know it's minor but probably better to leave everything as entirely translatable. 😅